src/3d/qgs3dexportobject.cpp

```diff
@@ -92,6 +80,18 @@ void Qgs3DExportObject::setupTextureCoordinates( const QVector<float> &texturesB
   mTexturesUV << texturesBuffer;
 }
 
+void Qgs3DExportObject::setupPhongMaterial( const QgsPhongMaterialSettings &material )
+{
+  QColor diffuse = material.diffuse();
+  QColor specular = material.specular();
+  QColor ambient = material.ambient();
+  float shininess = material.shininess();
+  setMaterialParameter( QString( "Kd" ), QString( "%1 %2 %3" ).arg( diffuse.redF() ).arg( diffuse.greenF() ).arg( diffuse.blueF() ) );
```

Comment:
```suggestion
  setMaterialParameter( QStringLiteral( "Kd" ), QStringLiteral( "%1 %2 %3" ).arg( diffuse.redF() ).arg( diffuse.greenF() ).arg( diffuse.blueF() ) );
```

For const rvalue strings it's better and faster to use `QStringLiteral` instead of `QString`, can you please check other occurrences too?

Reply:
Okay, thanks for the suggestion. I will try changing that whenever I notice myself doing it.
By the way should QStringLiteral be used for QString and const char* comparison too?